lib/Conversation/CommentForm/styles.scss

```diff
@@ -42,6 +42,21 @@ $comment-form-padding: $layout-spacing-base/2;
 /* ==========================================================================
    State Styles
    ========================================================================== */
-.has-value .comment-form__actions--submit {
-  display: inline-block;
-}
\ No newline at end of file
+   .new-conversation .is-submitted .button--secondary {
+     background-color: $neutral-base;
+     vertical-align: bottom;
+     padding-bottom: 0;
+   }
+   
+   .is-active .new-conversation .is-submitted {
+     .icon svg {
+       height:$layout-spacing-base;
+     }
+     .icon path {
+       fill: white;
+     }
+   }
+
+   .new-conversation .comment-form__input.is-submitted {
```

Comment:
Do we need the prefix of `.new-conversation` in the declaration?

I only ask because `.comment-form__input.is-submitted` feels like a nice state class that we only add when a certain scenario occurs, meaning that we shouldn't need additional classes in the declaration right?

It might just be confusing me due to the `is-submitted` class which I mention in some other feedback, maybe renaming that would clear it up.